docs/UsingNuGetPackages.md

```diff
@@ -0,0 +1,28 @@
+# Using NuGet Packages
+
+There are three options for using NuGet packages:
+
+1. Use `nuget_package` download a package from the internet.
```

Comment:
Should this one also be part of `import_nuget_package`, and this name be used for producing a nupkg?

Reply:
Alternatively, just give them 3 seperate names, all starting with `import_`?

Reply:
Hm, so my thought here is that I've mostly seen `import_` as a prefix (this isn't consistent... `java_import` uses it as a suffix ;) ) for "rule that brings in thing that exists on disk into Bazel". I imagine most people would use `nuget_package` and other rule sets have used the noun for their package type as the rule for downloading the package.

I don't think there is a lot of consistency though. e.g. `pip_import` actually downloads packages!

🤷‍♂